README.md

```diff
@@ -9,7 +9,11 @@ PolyKey requires a public/private keypair for all crypto operations, this can be
 The major dependencies of js-polykey are:
 - [kbpgp](https://github.com/keybase/kbpgp) for all asymmetric cryptography.
 - [EncryptedFS](https://github.com/MatrixAI/js-encryptedfs) for symmetric encryption within Vaults.
-- [isomorphi-git](https://github.com/isomorphic-git/isomorphic-git) for version control within Vaults.
+- [isomorphic-git](https://github.com/isomorphic-git/isomorphic-git) for version control within Vaults.
```

Comment:
I don't think we need to mention the deps here in the README.md. It's pretty clear what it is for from `package.json`.

Reply:
Sure, I will take them out.